### PR TITLE
Add simple Makefile

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	@echo
+	@echo "  - make install"
+	@echo
+	@echo "Test ramalama"
+	@echo
+	@echo "  - make test"
+	@echo
+	@echo "Clean the repository"
+	@echo
+	@echo "  - make test"
```
Comment (ericcurtin) [inline Makefile:19]: Typo "make clean", if this goes green before you get to it, I'll merge it anyway and catch in a follow on PR

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+build:
+	./container_build.sh
+
+.PHONY:
+test:
+	./ci.sh
+
+.PHONY: clean
+clean:
+	@find . -name \*~ -delete
```
Comment (ericcurtin) [inline Makefile:39]: I won't lie, I'm scared to run this, what does this do :smile:

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	@echo
+	@echo "  - make test"
+	@echo
+	@echo "Clean the repository"
+	@echo
+	@echo "  - make test"
+	@echo
+
+#
+# Create instructlab AI container images
```
Comment (ericcurtin) [inline Makefile:23]: Planning on putting us under the instructlab banner? :smile: Trying to read the tea leaves on this comment

File: Makefile (hunk 2 above)
Comment (ericcurtin) [reply Makefile:39]: Ok, deletes anything that ends in a ~, that seems perfectly fine

**Review by ericcurtin — APPROVED** [review]
Can catch typo in follow on PR if needs be

File: Makefile (hunk 2 above)
Comment (rhatdan) [reply Makefile:39]: Yes these are emacs backup files.

File: Makefile (hunk 3 above)
Comment (rhatdan) [reply Makefile:23]: oops copied from instructlab.

**Pull request merged**